### DISCUSSION TITLE
Iron Golem Despawn

### DISCUSSION THREAD

**Discussion by RVSkeLe** [PaperMC/Paper#10503] [Ideas]
### Is your feature request related to a problem?

Hello, we're currently having some issues due to the amount of iron golems in our players' farms. They have large quantities of Iron Golem Spawners and the Iron Golems do not despawn (contrary to Monsters).

### Describe the solution you'd like.

Add an option to make the iron golems despawn.

### Describe alternatives you've considered.

I've tried with plugins such as ClearLag but if players simply don't load the entities they won't get cleared, so the only actual option is the despawn of Iron Golems.

### Other

I haven't been able to find this option in the config, sorry if I missed it.

**Comment by YnsanitY16** [comment]
That's not a solution, i have this problem too
